lib/rules/email.js

```diff
@@ -18,6 +18,18 @@ module.exports = function({ schema, messages }, path, context) {
 		}
 	`);
 
+	src.push(`
+		var len = value.length;
+
+		if (len === 0) {
+			if (${schema.empty} === false) {
```

Comment:
@erfanium Shouldn’t this be the behaviour? Is the same for `string` for example, if `schema.empty` is undefined it will accept empty values.

Also if the condition is like you described, it means that if the `schema.empty` is NOT `true` but the value is, the error will be `Invalid email` instead of `Empty string`.